app/services/rad_common/date_filter.rb

```diff
@@ -0,0 +1,41 @@
+module RadCommon
+  class DateFilter
+    attr_reader :column
+    def initialize(column:, type:)
+      @column = column
+    end
+
+    def filter_view
+      'date'
+    end
+
+    def searchable_name
+      [start_input, end_input]
+    end
+
+    def start_at_value(params)
```

Comment:
Are start_at_value and end_at_value needing to be public? I can't easily tell. And apply_filter?

Reply:
Fixing, start_at_value and end_at_value are private but apply_filter needs to be public